Building without SSL support (USE="-ssl -gnutls") leads to a build
failure because of misplaced #ifdefs in code.  Fixed for versions
greater than 3.7.9 in upstream repository.

Gentoo bug: https://bugs.gentoo.org/show_bug.cgi?id=373611
Upstream bug: http://www.thewildbeast.co.uk/claws-mail/bugzilla/show_bug.cgi?id=2462

--- src/wizard.c
+++ src/wizard.c
@@ -1253,8 +1253,8 @@
 	GtkWidget *hbox;
 	GtkWidget *hbox_spc;
 	GtkWidget *smtp_auth_table;
-#ifdef USE_GNUTLS
 	GtkWidget *label;
+#ifdef USE_GNUTLS
 	GtkWidget *button;
 	GtkWidget *smtp_cert_table;
 #endif
